src/components/user-defined-sets/SetsManager.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import CurrentSetManager from './CurrentSetManager';
+
+export default function SetsManager(props) {
+  const {
+    sets,
+    onUpdateSets = (msg) => {
+      console.warn(`onUpdateSets from SetsManager ${msg}`);
+    },
+  } = props;
+  return (
+    <div className="sets-manager">
+      <CurrentSetManager sets={sets} onUpdateSets={onUpdateSets} />
+      <div className="set-list">
+        {Array.from(sets.namedSets.keys()).map(key => (
```

Comment:
`mapKeys`? Or since you use the value as well, `mapEntries`? https://immutable-js.github.io/immutable-js/docs/#/OrderedMap/mapEntries